core/modules/widgets/link.js

```diff
@@ -153,7 +153,7 @@ LinkWidget.prototype.handleDragStartEvent = function(event) {
 			}
 			// Then the data
 			dataTransfer.clearData();
-			var jsonData = this.wiki.getTiddlerAsJson(this.to),
+			var jsonData = this.wiki.getTiddlerAsJson(this.to,true),
```

Comment:
I don't like the 'exporting' parameter for getTiddlerAsJson. Wouldn't it be clearer to add `jsonData["_origin"] = window.location.href` here?